megamek/src/megamek/common/icons/AbstractIcon.java

```diff
@@ -109,14 +111,16 @@ public Image getImage(int size) {
         return getImage(size, -1);
     }
 
+    public Image getImage(int width, int height) {
+        return getImage(getBaseImage(), width, height);
+    }
+
     /**
      * This is used to create the proper image and scale it if required. It also handles null protection
      * by creating a blank image if required.
      * @return the created image
      */
-    public Image getImage(int width, int height) {
-        Image image = getBaseImage();
-
+    protected Image getImage(Image image, int width, int height) {
```

Comment:
If this can return `null`, as the above calls make it appear, lets add an `@Nullable` annotation.